### PR TITLE
Clean urls

### PR DESCRIPTION
**Pull request by dmitrydao**
URLs check via the linkcheck tool.
A general check gives the `Perfect` score, a check for external links is not perfect because the tool throws some errors for Remix and Github links. Not sure if we can even fix this.

### CODE REVIEW

File: docs/Additional Information/glossary.md [hunk 1]
```diff
@@ -6,21 +6,21 @@ permalink: "docs/glossary/"
...
-- [HTTPGet](/docs/adapters/#httpget)
-- [HTTPPost](/docs/adapters/#httppost)
-- [JSONParse](/docs/adapters/#jsonparse)
-- [Multiply](/docs/adapters/#multiply)
-- [NoOp](/docs/adapters/#noop)
-- [NoOpPend](/docs/adapters/#nooppend)
-- [Sleep](/docs/adapters/#sleep)
+An Adapter is a piece of software responsible for executing a specific piece of functionality. A Chainlink node comes with a number of Adapters built in, commonly known as Core Adapters, but can also be extended via [Bridges](/docs/node-operators) to connect with user defined [External Adapters](./#external-adapter). Core Adapters offered by the Chainlink Node by default:
+
+- [Bridge](/docs/node-operators)
```
Comment (StephenFluin) [inline docs/Additional Information/glossary.md:11]: Should this end with a trailing slash?

File: docs/Introduction/architecture-overview/architecture-decentralized-model.md [hunk 2]
```diff
@@ -92,12 +92,12 @@ Learn how to create a consumer contract to [Get the Latest Price](../get-the-lat
 
 Proxy contracts are on-chain proxies that store the most up-to-date Aggregator for a particular price feed. Using proxies enables the underlying Aggregator to be upgraded without any interruption of service for consuming contracts.
 
-See the <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
+See the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol" target="_blank">`AggregatorProxy`</a> contract on Github.
```
Comment (StephenFluin) [inline docs/Introduction/architecture-overview/architecture-decentralized-model.md:95]: Shouldn't we still point at 0.6?

File: docs/Introduction/architecture-overview/architecture-decentralized-model.md [hunk 3]
```diff
@@ -92,12 +92,12 @@ Learn how to create a consumer contract to [Get the Latest Price](../get-the-lat
...
 
 ## Aggregators
 
 Aggregators are the contracts that receive periodic price updates from multiple Oracles. They aggregate and store the current price on-chain so that consumers can obtain the latest price and act upon it within the same transaction.
 
-This data can be accessed by referencing the Price Feed address using the <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> contract.
+This data can be accessed by referencing the Price Feed address using the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> contract.
```
Comment (StephenFluin) [inline docs/Introduction/architecture-overview/architecture-decentralized-model.md:101]: 0.6 or 0.7 or 0.8?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Minor nits, looking great overall!

File: docs/Introduction/architecture-overview/architecture-decentralized-model.md (hunk 2 above)
Comment (dmitrydao) [reply docs/Introduction/architecture-overview/architecture-decentralized-model.md:95]: Yes but there's not AggregatorProxy for 0.6 that's why I picked 0.7. I can change to another interface if needed.

File: docs/Introduction/architecture-overview/architecture-decentralized-model.md (hunk 3 above)
Comment (dmitrydao) [reply docs/Introduction/architecture-overview/architecture-decentralized-model.md:101]: For now, it should be 0.6. I'll move it to 0.8 within the PR that moves code samples to 0.8.
But agree, this one should be 0.6. Thanks

File: docs/Introduction/architecture-overview/architecture-decentralized-model.md (hunk 2 above)
Comment (StephenFluin) [reply docs/Introduction/architecture-overview/architecture-decentralized-model.md:95]: Hrm. Can you try to figure out why we removed the 0.6 contract? That's unusual right?

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**